lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -355,6 +384,26 @@ _decodeSignedTx bytes = do
         Left apiErr ->
             Left $ ErrDecodeSignedTxWrongPayload (Cardano.renderApiError apiErr)
 
+_mkStdByronTx
+    :: WalletKey k
+    => ProtocolMagic
+    -> (Address -> Maybe (k 'AddressK XPrv, Passphrase "encryption"))
+    -> [(TxIn, TxOut)]
+    -> [TxOut]
+    -> Either ErrMkTx (Tx, SealedTx)
+_mkStdByronTx pm keyFrom ownedIns outs = do
+    let (Cardano.TxUnsignedByron _ _ txHash) =
+            mkUnsignedByronTx ownedIns outs
+    let tx = (fst <$> ownedIns, outs)
+    let sigData = Byron.hashToBytes txHash
+    witnesses <- forM ownedIns $ \(_, TxOut addr _) -> do
+        ks <- lookupPrivateKey keyFrom addr
+        pure $ mkByronWitness pm sigData ks
+    pure
+        ( Tx (Hash sigData) (second coin <$> ownedIns) outs
+        , SealedTx $ CBOR.toStrictByteString $ CBOR.encodeSignedTx tx witnesses
```

Comment:
I don't think this is right. This is how transaction were serialized in Byron, but in Shelley, they are following a different scheme. Therefore, we should serialize transaction using the new shelley methods; only the witnesses are different!